src/breaking-change.ts

```diff
@@ -159,7 +159,7 @@ export async function runScript() {
 
     const resolved = resolvedMapForNewSpecs[swagger]
     if (resolved) {
-      const diffs = await runOad(swagger, resolved);
+      const diffs = await runOad(path.resolve(pr!.workingDir,swagger), resolved);
```

Comment:
a minor formatting issue, a space is needed between `,` and `swagger`

Reply:
There is a space here in my local env.